src/DotLiquid.Tests/Tags/IncludeTagTests.cs

```diff
@@ -224,5 +252,19 @@ public void TestIncludeFromTemplateFileSystem()
             }
             Assert.AreEqual(fileSystem.CacheHitTimes, 1);
         }
+
+        public void WithFileSystem(IFileSystem fs, Action action)

```

Comment:
If we need this we should probably move to the `Helper` class and do a proper lock like there is in `LockTemplateStaticVars`